EIPS/eip-7848.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 7848
+title: On-chain upgrade signaling
+description: Allows participants to indicate readiness for a client upgrade when producing blocks
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-on-chain-upgrade-signaling/22306
```

Comment:
url needs fixing as it still has eip-xxxx

Reply:
0204d1b926edcd2fa400599b7a1b0d7ac010ed77